build.gradle.kts

```diff
@@ -23,6 +23,9 @@ dependencies {
     // runelite has outdated version with CVEs
     implementation(group = "com.google.guava", name = "guava", version = "31.1-jre")
 
+    // avoid excluding jetbrains annotations
+    compileOnly(group = "org.jetbrains", name = "annotations", version = "23.0.0")
```

Comment:
this compile dependency is safe to include: https://github.com/runelite/plugin-hub/blob/master/package/verification-template/build.gradle#L21